src/ipsis/course-merge/BaseCollectionEntity.js

```diff
@@ -62,5 +45,16 @@ export class BaseCollectionEntity extends Entity {
 	updateEntity(entity) {
 		this._entity = entity;
 	}
+
+	loadMorePageSize(collectionGetter) {
+		const pageSize = 20;
+		const totalCount = this.totalCount() ?? 0;
+		const collectionLength = collectionGetter?.length ?? 0;
```

Comment:
Your getters are functions
```suggestion
		const collectionLength = collectionGetter()?.length ?? 0;
```